.github/workflows/beam_PostCommit_Sickbay_Python.yml

```diff
@@ -52,27 +52,29 @@ env:
 
 jobs:
   beam_PostCommit_Sickbay_Python:
-    name: ${{matrix.job_name}} (${{matrix.job_phrase}} ${{matrix.python_version}})
+    name: ${{matrix.job_name}} (${{matrix.job_phrase_1}}${{matrix.job_phrase_2}})
```

Comment:
Should we include the python version here as well?

Reply:
It's done.
Thank you.